ci-operator/config/openshift/kubernetes-autoscaler/openshift-kubernetes-autoscaler-master.yaml

```diff
@@ -65,3 +65,18 @@ tests:
     make test-e2e
   openshift_installer_src:
     cluster_profile: aws
+- as: git-history
+  commands: |
+    PREFIX="UPSTREAM: <carry>: openshift:"
```

Comment:
Can you do the same (i.e. `PREFIX="UPSTREAM: <[[:alnum:]]+>: openshift:"`) as in https://github.com/openshift/release/pull/3116/files#diff-f0e4e08ecb398ef3dfe3a832e3714cd7R59 ?

Reply:
Did it locally and forgot to push changes :man_facepalming: